specification/sql/resource-manager/Microsoft.Sql/preview/2020-08-01-preview/DatabaseExtensions.json

```diff
@@ -218,27 +219,22 @@
           "x-ms-mutability": [
             "create"
           ]
-        },
-        "administratorLogin": {
-          "description": "Administrator login name.",
-          "type": "string",
-          "x-ms-mutability": [
-            "create"
-          ]
-        },
-        "administratorLoginPassword": {
-          "description": "Administrator login password.",
-          "type": "string",
-          "x-ms-mutability": [
-            "create"
-          ]
-        },
-        "authenticationType": {
-          "description": "Authentication type.",
-          "type": "string",
-          "x-ms-mutability": [
-            "create"
-          ]
```

Comment:
are these 3 properties new added in this version?

Reply:
Hello, this API is not being used by the customers yet. One of the main use cases for this API is creating a new datawarehouse (SQL Pool) from sample. Because this API was not ready during GA time, Synapse side feature was checked in behind a feature switch. Hence, this is not being used yet.

These properties, when I first designed this API, I was told that the SQL Import/Export team would also be using this API or modifying this API. These fields were added to support that. But now they aren't going that route now. So now this API, database Extensions, is used for polybase import only, and we don't support these. We have internal checks to make sure that these are not being passed.

Another change in this PR is I have changed the DatabaseExtensionsDefinition to DatabaseExtension Proxy Resource, as this is going to be used in ARM templates, and that needs the request body parameters to be enclosed within the properties tag. 

Hence, this breaking change is required.

Reply:
Ok, Thanks, once one of reviewer in SQL team like @ericshape approved the PR, I will merge it.